<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recipes</class>
 <widget class="QDialog" name="recipes">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>recipes</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="dataTable" name="tableWidget_recipes">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>modinfo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>nID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>strName</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_export">
         <property name="text">
          <string>export recipes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>modinfo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_modinfo">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>nID</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_nID">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>strName</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_strName">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>strSecretName</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_strSecretName">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>tools&amp;consumed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="dataTable" name="tableWidget_tools">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>modinfo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>nID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>strPropertyName</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>num</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>items</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="dataTable" name="tableWidget_items">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>modinfo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>id</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>nGroupID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>nSubgroupID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>strName</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>dataTable</class>
   <extends>QTableWidget</extends>
   <header>dataTable</header>
   <slots>
    <slot>add_line()</slot>
    <slot>copy_line()</slot>
    <slot>delete_line()</slot>
    <slot>find_text(QString)</slot>
    <slot>show_tooltips(int,int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>tableWidget_tools</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>recipes</receiver>
   <slot>show_item(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>805</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_export</sender>
   <signal>clicked()</signal>
   <receiver>recipes</receiver>
   <slot>export_recipes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>855</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget_tools</sender>
   <signal>cellActivated(int,int)</signal>
   <receiver>recipes</receiver>
   <slot>show_item(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>841</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget_recipes</sender>
   <signal>cellActivated(int,int)</signal>
   <receiver>recipes</receiver>
   <slot>show_recipe(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>-40</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tableWidget_recipes</receiver>
   <slot>find_text(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget_recipes</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>recipes</receiver>
   <slot>show_recipe(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>810</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_recipe(int,int)</slot>
  <slot>show_item(int,int)</slot>
  <slot>export_recipes()</slot>
 </slots>
</ui>
